cypress/e2e/shared/simpleTable.test.ts

```diff
@@ -53,7 +53,7 @@ describe('simple table interactions', () => {
     // verify correct number of pages
     cy.getByTestID('pagination-item')
       .last()
-      .contains('50')
+      .contains('100')
```

Comment:
we need to change the tests because when heights change, the simple table renders the cells on different pages.